Fix "hard" indent column info on status view.

dist-ian opened the pull request.

Guldoman reviewed (commented):
This might get slow on very long lines, as you're iterating for each character.

What do you think about something like:
```lua
      local _, indent_size = dv.doc:get_indent_info()
      local ntabs = 0
      local last_idx = 0
      while last_idx < col do
        local s, e = string.find(dv.doc.lines[line], "\t", last_idx, true)
        if s and s < col then
          ntabs = ntabs + 1
          last_idx = e + 1
        else
          break
        end
      end
      col = col + ntabs * (indent_size - 1)
```

I also think that we shouldn't limit this to only hard indented files.

I wonder if we should show both the col number and the "byte" number like `37:10(8)`.

dist-ian commented:
yes, i've changed it now and added a "byte" number with an option to show or hide it. But the tool tip will always show the info " (byte)", because it can't be changed without restart (or i don't know how to refresh the status bar items when configurations are changed via the user module.).

Guldoman commented:
Ah I realize I wasn't really clear: I meant showing the byte *position*, not the char byte value, to basically keep the old value if someone wanted it.

A way to manipulate the tooltip could be something like:
```lua
  local line_col_item
  line_col_item = self:add_item({
  ...
    get_item = function()
  ...
      if ntabs > 0 then
        line_col_item.tooltip = "line : column (byte)"
      else
        line_col_item.tooltip = "line : column"
      end
  ...
```
but I realize this is a bit hacky, so let's just avoid doing the bytes thing.
